README.md

```diff
@@ -1,78 +1,29 @@
-```
-
-* Hints on functionality to implement:
-  * Ensure you have a list of dishes with prices
-  * Place the order by giving the list of dishes, their quantities and a number that should be the exact total. If the sum is not correct the method should raise an error, otherwise the customer is sent a text saying that the order was placed successfully and that it will be delivered 1 hour from now, e.g. "Thank you! Your order was placed and will be delivered before 18:52".
-  * The text sending functionality should be implemented using Twilio API. You'll need to register for it. It’s free.
-  * Use the twilio-ruby gem to access the API
-  * Use the Gemfile to manage your gems
-  * Make sure that your Takeaway is thoroughly tested and that you use mocks and/or stubs, as necessary to not to send texts when your tests are run
-  * However, if your Takeaway is loaded into IRB and the order is placed, the text should actually be sent
-
-* Advanced! (have a go if you're feeling adventurous):
-  * Implement the ability to place orders via text message.
-
-* A free account on Twilio will only allow you to send texts to "verified" numbers. Use your mobile phone number, don't worry about the customer's mobile phone.
-* Finally submit a pull request before Monday at 9am with your solution or partial solution.  However much or little amount of code you wrote please please please submit a pull request before Monday at 9am
-
-
-In code review we'll be hoping to see:
-
-* All tests passing
-* High [Test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) (>95% is good)
-* The code is elegant: every class has a clear responsibility, methods are short etc. 
-
-Reviewers will potentially be using this [code review rubric](docs/review.md).  Referring to this rubric in advance will make the challenge somewhat easier.  You should be the judge of how much challenge you want this weekend.
-
-Notes on Test Coverage
-------------------
-
-You can see your [test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) when you submit a pull request, and you can also get a summary locally by running:
-
+------------
 ```
-$ coveralls report
+  2.2.3 :002 > chinese = TakeAway.new
+   => #<TakeAway:0x007ffefb0771f8 @order=#<Order:0x007ffefb0771d0 @basket={}, @current_order="", @total_amount=0, @menu={"Salted Chilli Ribs"=>5.1, "Spring Roll"=>1.5, "Prawn Toast"=>2.5, "Scallops With Ginger & Spring Onion"=>5.6, "Chicken in Blackbean"=>4.8, "Egg Fried Rice"=>1.9}>, @env={"account_sid"=>"account_sid", "auth_token"=>"auth_token", "from_number"=>"from", "to_number"=>"to"}> 
+  2.2.3 :003 > chinese.order "Egg Fried Rice", 4
+   => 7.6 
+  2.2.3 :004 > chinese.order "Spring Roll"
+   => 9.1 
+  2.2.3 :005 > chinese.basket_summary
+   => "Egg Fried Rice x4: £7.60\nSpring Roll x1: £1.50\n" 
+  2.2.3 :006 > chinese.total
+   => "Total: £9.10" 
+  2.2.3 :007 > chinese.complete_order(9.10)
 ```
```

Comment:
You should add instructions on handling env vars on your README file
https://github.com/makersacademy/takeaway-challenge/blob/master/docs/review.md#personal-details-and-tokens-on-github